core/tm/interfaces/IOrganizacion.ts

```diff
@@ -45,6 +45,12 @@ export interface IOrganizacion {
      * @memberof IOrganizacion
      */
     showMapa?: boolean;
+    configuraciones?: {
+        emails: [{
+            email: String,
+            nombre: String
+        }]
+    };
     esCOM: Boolean;
```

Comment:
Al menos en `typescript` se prefiere usar el tipo `string` por sobre el wrapper `String`. Lo mismo con Boolean/boolean, Number/number... 
```suggestion
            email: string,
            nombre: string
        }]
    };
    esCOM: boolean;
```